src/App/Game/Game.tsx

```diff
@@ -112,13 +113,13 @@ export default class Game extends React.Component<IGameProps, {}> {
           gameArgs,
         }),
       };
-      if (state.config.enhancers) {
-        const enhancers = [];
-        for (const enhancer of state.config.enhancers) {
-          enhancers.push(enhancer(gameArgs));
-        }
-        clientConfig.enhancer = applyMiddleware(...enhancers);
+      const allEnhancers = state.config.enhancers ?
+        state.config.enhancers.concat(DEFAULT_ENHANCERS) : DEFAULT_ENHANCERS;
+      const enhancers = [];
+      for (const enhancer of allEnhancers) {
+        enhancers.push(enhancer(gameArgs));
```

Comment:
replace this with const enhancers = allEnhancers.map((enhancer) => enhancer(gameArgs));